docs/rules/computed-property-spacing.md

```diff
@@ -48,6 +48,10 @@ obj[foo ]
 obj[ 'foo']
 var x = {[ b ]: a}
 obj[foo[ bar ]]
+
+// applies to the spacing for dynamic keys (computed property key) when destructuring objects.
```

Comment:
```suggestion
```

I think this comment is not necessary for the examples, since the other examples don't have it. We'll add an explanation in the opening section in https://github.com/eslint/eslint/pull/14004.